### PR TITLE
ADR-789: Task list page scenarios added

### PR DESCRIPTION
**Pull request by chiransan01**

### CODE REVIEW

File: src/test/resources/features/AlcoholDutyReturnsTaskListPage.feature [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      | Name     | Alcohol strength | Tax type code | Volume     | Pure alcohol | Duty rate        | Duty due |
+      | TestName | 9% ABV           | 333           | 100 litres | 9 litres     | £28.50 per litre | £256.50  |
+    When I click save and continue button on "Check Your Answers Page"
+    Then I am presented with the "Product List Page"
+    When I select radio button "No" on "Product List Page"
+    And I click save and continue button on "Product List Page"
+    Then I am presented with the "Task List Page"
+    And I should see the following status of the submission journey
+      | You’ve told us you need to declare duty | Change the products you’ve told us about |
+      | Completed                               | Completed                                |
```
Comment (Kavitakum) [inline src/test/resources/features/AlcoholDutyReturnsTaskListPage.feature:111]: Is it not possible to add this scenario as well here in tests?:

Given I have submitted at least one product entry screen through the returns journey,

And I haven't selected the 'No' radio button on the 'Product list' screen,

And navigate away from the service,

When I navigate back to the 'Task list' screen,
Then the status for ‘Change the products you've told us about' hyperlink should change to 'In progress'.

Comment (chiransan01) [reply src/test/resources/features/AlcoholDutyReturnsTaskListPage.feature:111]: Once the status is changed to 'Completed' we can't change it, and this needs a new journey. If you look at scenario 3 at lines 71-73, this has been addressed.

Comment (Kavitakum) [reply src/test/resources/features/AlcoholDutyReturnsTaskListPage.feature:111]: Thanks..Not sure how it was missed by me :-)

**Review by Kavitakum — APPROVED** [review]
Approved

**Pull request merged**